main.tex

```diff
@@ -1963,9 +1926,7 @@
 
 \item  Despite any other Rules, for 5 years after the date of registration of the Plan of Subdivision, the Developer or any agent of the Developer may erect:
 
-(i) ‘for sale’ signs or other advertising signs; or
```

Comment:
This is inconsistent with the rules about advertising signs erected by other owners.